pkg/metrics/collector.go

```diff
@@ -0,0 +1,84 @@
+package metrics
+
+import (
+	"maps"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/aquasecurity/tracee/pkg/counter"
+	"github.com/aquasecurity/tracee/pkg/logger"
+)
+
+type Collector struct {
+	m           sync.RWMutex
+	description string
+	values      map[uint64]uint64
```

Comment:
@NDStrahilevitz I have fixed this key/value, but perhaps they can be generic. WDYT?

Reply:
Already made it generic.